### DISCUSSION TITLE
Support for Epic Games version

### DISCUSSION THREAD

**Discussion by JoenvandeVorle** [Shazbot/WH3-Mod-Manager#61] [Ideas]
I play via the epic games version, and while I have put the correct file path to warhammer3.exe, the mod manager just opens the CAlauncher for me instead of the game. This results in all of my mods getting disabled. Not sure what is going but maybe look into adding support for the epic games version of WH3 specifically.
Of course I'm willing to help in any way. For now I solved it by just copying the steam workshop folder (I downloaded all workshop mods via steamcmd) to Warhammer3/Mods so I can launch them via the CA launcher

**Comment by EarlofLocksley** [comment]
JoenvandeVorle, I also have an EPIC games TW:WH3. How did you get steam mods downloaded for your game? Even using the steamcmd as an attempted work around has not worked for me and it seems like there is very little in the way of non-steam mods for this game. Especially mods that help other moders.

**Reply by JoenvandeVorle** [reply]
Luckily I own WH3 on steam as well as on epic (I just play on epic cus I have DLC there), so I was able to use steamcmd. However if you don't own the game on steam you can't use steamcmd to download workshop stuff for it sadly. You can try looking on some sites like [skymods](https://catalogue.smods.ru/game/total-war-warhammer-iii/) for (re)uploads of the same mods that are on the workshop, but other than that there are not many sources, as most of the modding seems to happen via the workshop
